sdk/android/ooyalaSkinSDK/src/main/java/com/ooyala/android/ooyalaskinsdk/OoyalaSkinLayoutController.java

```diff
@@ -181,9 +181,15 @@ public void setFullscreenButtonShowing(boolean showing) {
 
   @ReactMethod
   public void onClosedCaptionUpdateRequested(ReadableMap parameters) {
-    final String languageName = parameters.hasKey("language") ? parameters.getString("language") : null;
-    double curTime = _player.getPlayheadTime() / 1000d;
-    WritableMap params = BridgeMessageBuilder.buildClosedCaptionUpdateParams(_player, languageName, curTime);
+    WritableMap params;
+    // ignore the request if cc is not available
+    if (_player == null || _player.getCurrentItem() == null || !_player.getCurrentItem().hasClosedCaptions()) {
+       params = Arguments.createMap();
```

Comment:
not sending response is my first choice, but I found the last cc of previous video will stay there so I send an empty one to clear it. The alternative way is to reset the state when embedcode is set but if next video also have cc then user won't be able to see it and have to select it again.